api/contract/cli.yml

```diff
@@ -229,6 +225,10 @@ components:
       $ref: "./openapi/src/cloud/schemas/MeasurementSchemaUpdateRequest.yml"
     Query:
       $ref: "./overrides/schemas/Query.yml"
+    InfluxqlCsvResponse:
+      $ref: "./overrides/schemas/InfluxqlCsvResponse.yml"
```

Comment:
This should be in the openapi repo rather than `overrides` I think - we want to shrink the size of `overrides` over time rather than grow it.

Reply:
I think we should have a sync conversation then with the three of us - I don't want to pull you in multiple directions.